setup.py

```diff
@@ -75,7 +75,60 @@
 # no tests/docs in `src` so don't need exclude
 PACKAGES = find_packages(SOURCE_DIR)
 PACKAGE_DIR = {"": SOURCE_DIR}
-PACKAGE_DATA = {"openscm_runner": [os.path.join("adapters", "fair_adapter", "*.csv")]}
+PACKAGE_DATA = {
+    "openscm_runner": [
+        os.path.join("adapters", "fair_adapter", "*.csv"),
+        os.path.join("adapters", "ciceroscm_adapter", "utils_templates", "*.txt"),
```

Comment:
Is it possible to pull this out i.e. let the user have CICERO somewhere on their machine and we add an environment variable to point the right place rather than including CICERO as part of the package?

Reply:
I guess, but I'm not sure I'd know how to make that work on my own setup...

Reply:
Ok depending on the answers to the other questions I can make a PR for this

Reply:
Let's leave it for now, we can talk about how to setup the model to make it a bit more user friendly another time

Reply:
#49